### PR TITLE
Update FerretDB version

### PR DESCRIPTION
**Pull request by chilagrow**
https://github.com/FerretDB/engineering/issues/54

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,19 +19,19 @@ go run main.go
...
+Using MongoDB:          6.0.42
+Using Mongosh:          1.8.0
 
 For mongosh info see: https://docs.mongodb.com/mongodb-shell/
 
 ------
    The server generated these startup warnings when booting
-   2023-03-15T07:01:22.38Z: Powered by FerretDB unknown and PostgreSQL 15.2.
-   2023-03-15T07:01:22.38Z: Please star us on GitHub: https://github.com/FerretDB/FerretDB.
+   2023-04-04T01:09:24.993Z: Powered by FerretDB unknown and PostgreSQL 15.2.
```
Comment (chilagrow) [inline README.md:31]: Somehow it has `FerretDB unknown`. Anyone know why that is?

Comment (rumyantseva) [reply README.md:31]: I think, it should be a version.

Comment (chilagrow) [reply README.md:31]: There is no easy solution this for now.

**Review by rumyantseva — APPROVED** [review]
LGTM

**Review by AlekSi — APPROVED** [review]

**Pull request merged**